packages/dapp-svelte-wallet/api/src/lib-wallet.js

```diff
@@ -366,10 +293,16 @@ export async function makeWallet({
 
   function updateAllIssuersState() {
     issuersUpdater.updateState(
-      [...brandMapping.petnameToVal.entries()].map(([petname, brand]) => [
-        petname,
-        brandTable.get(brand),
-      ]),
+      [...brandMapping.petnameToVal.entries()].map(([petname, brand]) => {
+        const issuerRecord = brandTable.getByBrand(brand);
+        return [
+          petname,
+          {
+            issuerBoardId: issuerToBoardId.get(issuerRecord.issuer),
```

Comment:
I would prefer moving this line below the `...issuerRecord`, just to emphasise we're overriding it.

Reply:
Yup, sounds good